src/System.Private.CoreLib/shared/Interop/Unix/Interop.Libraries.cs

```diff
@@ -2,14 +2,12 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-
 internal static partial class Interop
 {
     internal static partial class Libraries
     {
         internal const string CoreLibNative = "System.Private.CoreLib.Native";
-        internal const string GlobalizationInterop = "System.Globalization.Native";
+        internal const string GlobalizationInterop = "System.Globalization.Native"; // CoreFX wrappers for ICU
```

Comment:
I would delete the `CoreFX wrappers for ICU` comment - it is misleading because of System.Globalization.Native does not live in CoreFX.